Plots/XY/NCL_tm_1.py

```diff
@@ -0,0 +1,120 @@
+"""
+NCL_tm_1.py
+===============
+Tickmark and Axis Manipulation example
+
+Note: This script is aimed at demonstrating the explicit handling of tick marks, their locations, labels, etc.;
+therefore, the use of geocat-viz convenience functions is minimized here to show such tick management functions directly
+throughout this script.
```

Comment:
I would suggest moving this to be the last part of the top docstring to be more consistent with previous example formatting.

Reply:
Hmm. I copied the format of tm_2 that looks like this.

"""
NCL_tm_2.py
===============
Tickmark and Axis Manipulation example

Note: This script is aimed at demonstrating the explicit handling of tick marks, their locations, labels, etc.;
therefore, the use of geocat-viz convenience functions is minimized here to show such tick management functions directly
throughout this script.

This script illustrates the following concepts:
   - Explicitly setting tickmarks and labels on the bottom X axis
   - Setting the spacing for tickmarks
   - Setting the mininum/maximum value of the Y axis in an XY plot
   - Changing the width and height of a plot

See following URLs to see the reproduced NCL plot & script:
    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/tm_2.ncl
    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/tm_1_1_lg.png and https://www.ncl.ucar.edu/Applications/Images/tm_1_2_lg.png
"""

Reply:
Huh, you're right about that. I would still remove the "Tickmark and Axis Manipulation example" but if you want to keep the Note above concepts illustrated, that's fine with me.